clients/go/pkg/auditopt/config.go

```diff
@@ -260,13 +260,11 @@ func labelsFromConfig(cfg *api.Config) audit.Option {
 }
 
 func justificationFromConfig(ctx context.Context, cfg *api.Config) (audit.Option, error) {
-	// TODO(#299): allow overriding these values via JVS env var.
-	jvsconfig := &client.JVSConfig{
-		JWKSEndpoint: cfg.Justification.PublicKeysEndpoint,
-		Version:      "1",
-		CacheTimeout: 5 * time.Minute,
+	var jvsconfig client.JVSConfig
+	if err := cfgloader.Load(ctx, &jvsconfig, cfgloader.WithEnvPrefix("JVS_")); err != nil {
```

Comment:
I thought we were dropping the `JVS_` prefix

Reply:
This is in the lumberjack _client_ (SDK) running in user's application code. We should add proper prefix to differentiate from other potential app configurations.